modules/ossm-release-2-3-12.adoc

```diff
@@ -0,0 +1,42 @@
+////
+Module included in the following assemblies:
+* service_mesh/v2x/servicemesh-release-notes.adoc
+////
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-version-2-3-12_{context}"]
+== {SMProductName} version 2.3.12
```

Comment:
```suggestion
[id="ossm-release-2-3-12_{context}"]
= {SMProductName} version 2.3.12
```

For all the modules, ID names should be the same as the filenames, and the first heading in the file should be a H1 (`=`)